src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -689,8 +693,10 @@ function restoreOriginal(questionNum) {
         $(`#${ParamsNames.FEEDBACK_QUESTION_DISCARDCHANGES}-${questionNum}`).hide();
         $(`#${ParamsNames.FEEDBACK_QUESTION_EDITTYPE}-${questionNum}`).val('');
         $(`#button_question_submit-${questionNum}`).hide();
-        $(`#questionnum-${questionNum}`).val(questionNum);
-        $(`#questionnum-${questionNum}`).prop('disabled', true);
+        $(`#${ParamsNames.FEEDBACK_QUESTION_NUMBER}-${questionNum}`).val(questionNum);
+        $(`#${ParamsNames.FEEDBACK_QUESTION_NUMBER}-${questionNum}`).prop('disabled', true);
+        $(`#${ParamsNames.FEEDBACK_QUESTION_NUMBER}-${questionNum}`).hide();
+        $(`#questionnum-non-menu-${questionNum}`).show();
```

Comment:
Now that there are two different views of question numbers (one with option, and one static), as compared to just one view in the past, you may want to consider renaming the two variables so that it is easy to tell which is which.